File: poo/Java/card.java
class Card extends Payment {
    String typeCard;
    Integer number; 
    Integer cvv;
    Integer date;

    public Card(Integer id, Integer amount, String typeCard, Integer number, Integer cvv, Integer date){
        super(id, amount);
        this.amount = amount;
        this.typeCard = typeCard;
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}

File: poo/PHP/card.php
<?php 
require_once('payment.php');
class Cash extends Payment{
    public $amount;
    public $typeCard;

    public function __construct($id, $amount, $typeCard) {
        parent::__construct($id);
        $this->amount = $amount;
        $this->typeCard = $typeCard;
    }
}
?>